admin/tools/docker/deployment/parallel/run.sh

```diff
@@ -63,13 +63,20 @@ else
     echo "No existing ${CONTAINER_NAME} container on any nodes"
 fi
 
+USE_JEMALLOC='/qserv/stack/stack/current/Linux64/jemalloc/master-g708711ce34/lib/libjemalloc.so'
```

Comment:
No idea how docker scripts work, but if you managed to do normal EUPS setup then this could probably be changed to `$JEMALLOC_DIR/lib/libjemalloc.so`